backend/src/test/repository/EntityIntegration.test.ts

```diff
@@ -294,4 +306,43 @@ describe('Integration tests for all entities', function () {
       fail('Should not have failed: ' + e);
     }
   });
+
+  test('Cascade delete - deleting a promotion will delete its schedules', async () => {
+    const promotion = promotions_sample[0];
+    assert(promotion.discount !== null);
+
+    // persist into db
+    await userRepository.save(promotion.user);
```

Comment:
Oh my only reason was because I don't expect the persisting to fail, this part is essentially saving the data so that it's there for our integration test to use.